index.html

```diff
@@ -1,90 +1,91 @@
-<!doctype html>
-
+<!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
   <title>Fun Bus Travel Agency - Solution</title>
-
-  <link href="https://fonts.googleapis.com/css?family=Indie+Flower|Roboto" rel="stylesheet">
-  <link href="css/index.css" rel="stylesheet">
-
-  <script src="js/index.js" async></script>
-
-  <!--[if lt IE 9]>
+  <link href="https://fonts.googleapis.com/css?family=Indie+Flower|Roboto"
+        rel="stylesheet">
+  <link href="css/index.css"
+        rel="stylesheet">
+  <script async
+        src="js/index.js"></script><!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
 <body>
-
   <header class="main-navigation">
     <div class="container nav-container">
-      <h1 class="logo-heading">Fun Bus</h1>
+      <h1 class="logo-heading"
+          id="logo-header">Fun Bus</h1>
       <nav class="nav">
-        <a class="nav-link" href="#">Home</a>
-        <a class="nav-link" href="#">About Us</a>
-        <a class="nav-link" href="#">Blog</a>
-        <a class="nav-link" href="#">Contact</a>
+        <a class="nav-link"
+             href="#">Home</a> <a class="nav-link"
+             href="#">About Us</a> <a class="nav-link"
+             href="#">Blog</a> <a class="nav-link"
+             href="#">Contact</a>
       </nav>
     </div>
   </header>
-     
   <div class="container home">
     <header class="intro">
-      <img src="img/fun-bus.jpg" alt="bus in the sand">
+      <img alt="bus in the sand"
+              onmouseout="this.src='img/fun-bus.jpg';"
```

Comment:
I would recommend using addEventListener rather than inline event listeners, as you have more control over them